modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -872,6 +872,65 @@ def attempt_convert_item(input_value, valid_classes, path_to_item,
     return input_value
 
 
+def is_type_nullable(input_type):
+    """
+    Returns true if None is an allowed value for the specified input_type.
+
+    A type is nullable if at least one of the following conditions is true:
+    1. The OAS 'nullable' attribute has been specified,
+    1. The type is the 'null' type,
+    1. The type is a anyOf/oneOf composed schema, and a child schema is
+       the 'null' type.
+    Args:
+        input_type (type): the class of the input_value that we are
+            checking
+    Returns:
+        bool
+    """
+    if input_type is none_type:
+        return True
+    # TODO: how do we handle the 'nullable: true' scenario?
+    if issubclass(input_type, ModelComposed):
+        # If oneOf/anyOf, check if the 'null' type is one of the allowed types.            
+        for t in input_type._composed_schemas.get('oneOf', ()):
+            if t is none_type:
+                return True
```

Comment:
t could also be a ModelComposed subclass.
How about also recursively calling is_type_nullable on t here?
- this also applies in the below anyOf block

Reply:
done